types/fundamental-react/lib/Tabs.d.ts

```diff
@@ -0,0 +1,36 @@
+import * as React from "react";
+
+export type TabProps = {
+    className?: string;
+    disabled?: boolean;
+    /* Icon to display on the tab. */
+    glyph?: string;
+    id?: string;
+    /* _INTERNAL USE ONLY._ */
+    index?: number;
+    /* Additional props to be spread to the tab\'s <a> element. */
+    linkProps?: { [x: string]: any };
+    /* _INTERNAL USE ONLY._ */
+    selected?: boolean;
+    /* Additional props to be spread to the tab content's <div> element. */
+    tabContentProps?: { [x: string]: any };
+    /* Localized text to display on the tab. */
+    title?: string;
+    /* _INTERNAL USE ONLY._ */
+    onClick?: (
+        event: React.MouseEvent<HTMLAnchorElement, MouseEvent>,
+        index?: number
```

Comment:
As discussed [here](http://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html), it's not recommended to have optional parameters in callbacks. The [underlying library](https://github.com/SAP/fundamental-react/blob/master/src/Tabs/Tab.js#L44) seems to always call this function with an `index` argument.

Reply:
that's great feedback! thank you :)